cmake/modules/RootMacros.cmake

```diff
@@ -641,7 +645,7 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
   # would not happen if we used target_sources() directly with the dictionary source.
   if(TARGET "${ARG_MODULE}" AND NOT "${ARG_MODULE}" STREQUAL "${dictionary}")
     add_library(${dictionary} OBJECT ${dictionary}.cxx)
-    set_target_properties(${dictionary} PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
+    set_target_properties(${dictionary} PROPERTIES POSITION_INDEPENDENT_CODE TRUE ARG_NO_MODULE ${ARG_NO_CXXMODULE})
```

Comment:
I didn't test this, but is it really `ARG_NO_MODULE` without `${...}` etc? Usually, this would be a variable if it's named `ARG_`. If it's a property, it would likely be named `NO_MODULE`.

Reply:
I am trying to set up a property named as ARG_NO_MODULE, should I rename it too NO_MODULE?